README.md

```diff
@@ -4,26 +4,30 @@
 ![Tensorflow](https://img.shields.io/badge/tensorflow-v2.9.0+-success.svg)
 [![Contributions Welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/keras-team/keras-cv/issues)
 
+# Vision
+A computer vision library dedicated for auto-driving, robotics and on device applications.
```

Comment:
Maybe we should workshop this vision statement a bit as a team?

A good mission statement would be useful, and the mission section below doesn't have one.

Reply:
This blurb below could be a good starting point for a mission statement, which should appear at the top of the mission section:

```
KerasCV's primary goal is to provide a coherent, elegant, and pleasant API to train state of the art computer vision models.
```

Reply:
can you elaborate? the vision statement is about "what is the ideal state for the product", in which case becoming the standard TF library for those applications seem most ideal (or we haven't come up with new applications that we think it should support)
Re mission statement -- yeah I agree this still needs to be improved

Reply:
yeah sgtm -- maybe we can workshop a mission statement in our next team sync

Reply:
sounds good!